### DISCUSSION TITLE
Symfony has disappeared from Techempower benchmark

### DISCUSSION THREAD

**Discussion by maxkain** [symfony/symfony#48117] [Q&A]
https://www.techempower.com/benchmarks/

"symfony", "symfony-swoole" had been there, but the they have disappeared. What is the reason?

**Comment by MatTheCat** [comment]
Seems like an outdated composer.json prevented Symfony to be in it: https://github.com/TechEmpower/FrameworkBenchmarks/issues/7289#issuecomment-1178126605
